src/site/apt/development_xmark_documentation.apt

```diff
@@ -0,0 +1,76 @@
+~~ Licensed to the Apache Software Foundation (ASF) under one or more
+~~ contributor license agreements.  See the NOTICE file distributed with
+~~ this work for additional information regarding copyright ownership.
+~~ The ASF licenses this file to You under the Apache License, Version 2.0
+~~ (the "License"); you may not use this file except in compliance with
+~~ the License.  You may obtain a copy of the License at
+~~
+~~     http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing, software
+~~ distributed under the License is distributed on an "AS IS" BASIS,
+~~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~~ See the License for the specific language governing permissions and
+~~ limitations under the License.
+
+
+XMark Benchmark Documentation
+    
+    This page displays VXQuery's performance on XMark Benchmark. In the following 
+    table, <Query Number> column represents the XMark query number. <Original Query> column 
```

Comment:
Add "in the following table, XMark queris are listed to show the current status of each query in VXQuery. the Query Number ...."